lib/github.js

```diff
@@ -143,6 +143,32 @@ class GitHub {
     })
   }
 
+  getPr (pr) {
+    const owner = this.config.owner
+    const repo = this.config.repo
+
+    return new Promise((resolve, reject) => {
+      const url = `https://api.github.com/repos/${owner}/${repo}/pulls/${pr}`
+      if (process.env.VERBOSE) {
+        console.log(`About to send GET to ${url}`)
+      }
+      fetch(url, getFetchOpts())
+        .then((resp) => {
+          if (!resp.ok) {
+            reject(`Error: ${resp.status}: ${JSON.stringify(resp.json, null, 2)}`)
+            return
+          }
+
+          resp.json().then((json) => {
+            resolve(json)
+          })
+        })
+        .catch((err) => {
+          reject(err)
+        })
+    })
+  }
```

Comment:
You are absolutely correct, this was a copy-paste-reduce scenario where I should have further cleaned it up.